src/main/kotlin/no/nb/mlt/wls/product/payloads/ApiProductPayload.kt

```diff
@@ -23,7 +23,7 @@ data class ApiProductPayload(
     @Schema(example = "1.0")
     val quantity: Float,
     @Schema(example = "NONE")
-    val preferredEnvironment: PreferredEnvironment,
+    val environment: Environment,
```

Comment:
This should still be called preferredEnvironment in the outward facing API, we can just call the enum Environment for ease of use and since we will likely re-use it when we start storing information about preferred env and actual storage env.  
Sorry for not making myself clear here.